src/lib/components/forms/form/example/form-fields.ts

```diff
@@ -105,6 +105,26 @@ export const containerCompontent: TerraFormFieldInterface = {
 };
 
 export const formFields: TerraKeyValueInterface<TerraFormFieldInterface> = {
+    suggestion: {
+        type: 'suggestion',
+        defaultValue: 'suggestion1',
+        options: {
+            name: 'Suggestion',
+            required: false,
+            listBoxValues: [
+                {
+                    value: 'suggestion1',
+                    caption: 'Hello',
+                    icon: 'icon-add'
+                },
+                {
+                    value: 2,
+                    caption: 'World',
+                    icon: 'icon-delete'
+                }
+            ]
+        }
+    },
```

Comment:
Es gibt bereits ein formField mit type suggestion. Siehe Zeilen 34 - 50 in dieser Datei. Ggf. veränderst du dieses und fügst es hier hinzu.